stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,433 @@
+            </div>     
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+placeHolderImage.storyName = 'Placeholder User Image';
+placeHolderImage.parameters = {
+    docs: {
+        storyDescription: 'The Feed list item with placeholder user image'
+    }
+};
+
+export const withoutUserImage = () => `
+    <div class="fd-feed-list" aria-label="Feed List Item without user image example">
+        <div class="fd-feed-list__body">     
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+withoutUserImage.storyName = 'Without User Image';
+withoutUserImage.parameters = {
+    docs: {
+        storyDescription: 'The Feed list item without user image'
+    }
+};
+
+export const withUserLink = () => `
+    <div class="fd-feed-list" aria-label="Feed List Item with user link example">
+        <div class="fd-feed-list__body">
+            <div 
+                class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list--link fd-feed-list__thumb"  
+                style="background-image: url(http://lorempixel.com/460/620/people/7/)"
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe"
+            ></div>
+                 
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">
+                        <a href="#" class="fd-link fd-link--emphasized" tabindex="0">Joe Doe: </a> 
+                    </span>
+                    Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+withUserLink.storyName = 'With User link';
+withUserLink.parameters = {
+    docs: {
+        storyDescription: 'The Feed list item with link'

```

Comment:
The feed list item can display linked usernames (and images). To display a linked username, add the `fd-link` class with the `fd-link--emphasized` modifier to the `fd-feed-list__name` element.